cmd/skywire-cli/commands/visor/root.go

```diff
@@ -27,17 +25,11 @@ var RootCmd = &cobra.Command{
 }
 
 func rpcClient() visor.RPCClient {
+	const rpcDialTimeout = time.Second * 5
```

Comment:
Would it be enough for everything?

Reply:
This is only for dialing. A.k.a establishing a TCP connection with the visor (in this case).

Reply:
@evanlinjin Thank you for the explanation. It's fine then.